subsys/net/lib/http/http_server_http1.c

```diff
@@ -37,21 +37,19 @@ static const char conflict_response[] = "HTTP/1.1 409 Conflict\r\n\r\n";
 static const char final_chunk[] = "0\r\n\r\n";
 static const char *crlf = &final_chunk[3];
 
-static int handle_http1_static_resource(
-	struct http_resource_detail_static *static_detail,
-	struct http_client_ctx *client)
+static int handle_http1_static_resource(struct http_resource_detail_static *static_detail,
+					struct http_client_ctx *client)
 {
-#define RESPONSE_TEMPLATE			\
-	"HTTP/1.1 200 OK\r\n"			\
-	"%s%s\r\n"				\
+#define RESPONSE_TEMPLATE                                                                          \
+	"HTTP/1.1 200 OK\r\n"                                                                      \
+	"%s%s\r\n"                                                                                 \
 	"Content-Length: %d\r\n"
 
 	/* Add couple of bytes to total response */
 	char http_response[sizeof(RESPONSE_TEMPLATE) +
 			   sizeof("Content-Encoding: 01234567890123456789\r\n") +
 			   sizeof("Content-Type: \r\n") + HTTP_SERVER_MAX_CONTENT_TYPE_LEN +
-			   sizeof("xxxx") +
-			   sizeof("\r\n")];
+			   sizeof("xxxx") + sizeof("\r\n")];
```

Comment:
This is also cosmetic change. If you want to propose these, please have a separate commit or PR